friends/src/components/FriendList.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+import axios from 'axios'
+import Friend from './Friend'
+import PropTypes from 'prop-types';
+
+class FriendList extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            friends: [],
+        };
+    }
+
+    componentDidMount(){
+        axios
```

Comment:
Very good job GETing this data from the server, and setting state with the correct data!